docs/SPEC.md

```diff
@@ -311,9 +310,9 @@ To include the literal curly brackets within a string they must be escaped.
 Interpolation example:
 
     n = 42
-    "the answer is {n}" // the answer is 42
-    "the answer is not {n+1}" // the answer is not 43
-    "openinng curly bracket \{" // openinng curly bracket {
+    "the answer is ${n}" // the answer is 42
+    "the answer is not ${n+1}" // the answer is not 43
+    "opening curly bracket \${" // opening curly bracket {
```

Comment:
nit, typo at the end missing `$` line should be `"opening curly bracket \${" // opening curly bracket ${`